rocky/katalogus/views/boefje_setup.py

```diff
@@ -31,31 +60,139 @@ def get_context_data(self, **kwargs):
 
         return context
 
+
+class AddBoefjeVariantView(BoefjeSetupView):
+    """View where the user can create a Boefje variant, based on another Boefje."""
+
+    def setup(self, request, *args, **kwargs):
+        super().setup(request, *args, **kwargs)
+
+        self.plugin_id = str(uuid.uuid4())
+        self.return_to_plugin_id = self.kwargs.get("plugin_id")
+        katalogus = get_katalogus(self.organization.code)
+        self.plugin = katalogus.get_plugin(self.return_to_plugin_id)
+        consumes = []
+
+        for input_object in list(self.plugin.consumes):
+            consumes.append(input_object.__name__)
```

Comment:
You can iterate over sets like any other sequence, no need to create a list
```suggestion
        for input_object in self.plugin.consumes:
            consumes.append(input_object.__name__)
```